ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh

```diff
@@ -3,12 +3,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-echo "## Install yq"
-curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
-echo "   yq installed"
-
 /tmp/yq -i '.networking.networkType="OpenShiftSDN"' "${SHARED_DIR}/install-config.yaml"
```

Comment:
The `yq` from cli-yq image is located in `/tmp` ?

Reply:
Yes. As I want to make as less change as possible.

Reply:
You can see several jobs rely on this step already passed

Reply:
Added in https://github.com/openshift/release/pull/31245

Reply:
To save your time, it is
```
FROM ocp/4.5:cli
RUN curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```